.circleci/config.yml

```diff
@@ -140,7 +140,10 @@ jobs:
               sleep 1
             done
           background: true
-
```

Comment:
Nit: Rest of the file has whitespace between list entries. Think it makes it a bit more readable, but I don't have a strong preference on this, but if we remove whitespace here best remove it elsewhere in file for consistency.

Reply:
Do you mean the newlines between the steps of the job? Otherwise, that list entries are you talking about?

Reply:
Yes, that's what I meant, sorry for ambiguity.

Reply:
OK fixed it.